### PR TITLE
[FMT] apply black to plotting - part 2

### PR DESCRIPTION
**Pull request by Remi-Gau**
Relates to https://github.com/nilearn/nilearn/issues/2528

Changes proposed in this pull request:

- update configs to more from nilearn/plotting
- apply black

### CODE REVIEW

File: nilearn/plotting/tests/test_matrix_plotting.py [hunk 1]
```diff
@@ -21,25 +21,31 @@
...
+)
 def test_sanitize_figure_and_axes_error(fig, axes):
     from ..matrix_plotting import _sanitize_figure_and_axes
-    with pytest.raises(ValueError,
-                       match=("Parameters figure and axes cannot "
-                              "be specified together.")):
+
+    with pytest.raises(
+        ValueError,
+        match=("Parameters figure and axes cannot " "be specified together."),
```
Comment (Remi-Gau) [inline nilearn/plotting/tests/test_matrix_plotting.py:32]:
```suggestion
        match=("Parameters figure and axes cannot be specified together."),
```

File: nilearn/image/tests/test_image.py [hunk 2]
```diff
@@ -552,7 +552,7 @@ def test_index_img_error_4D(affine):
     ]:
         with pytest.raises(
             IndexError,
-            match="out of bounds|invalid index|out of range|" "boolean index",
+            match="out of bounds|invalid index|out of range| boolean index",
```
Comment (Remi-Gau) [inline nilearn/image/tests/test_image.py:555]:
```suggestion
            match="out of bounds|invalid index|out of range|boolean index",
```

File: nilearn/plotting/find_cuts.py [hunk 3]
```diff
@@ -443,15 +475,16 @@ def find_parcellation_cut_coords(labels_img,
...
+        left_hemi[int(x) :] = 0
+        right_hemi[: int(x)] = 0
 
         # Two connected component in both hemispheres
-        if (not np.all(left_hemi == False) or  # noqa: E712
-                np.all(right_hemi == False)):  # noqa: E712
-            if label_hemisphere == 'left':
+        if not np.all(left_hemi == False) or np.all(  # noqa: E712
+            right_hemi == False  # noqa: E712
+        ):
```
Comment (ymzayek) [inline nilearn/plotting/find_cuts.py:484]: I find this logic confusing, maybe flag to refactor (in another PR)

**Review by ymzayek — APPROVED** [review]
LGTM thx!

**Review by bthirion — APPROVED** [review]
LGTM, thx.

**Pull request merged**